src/main/java/bio/terra/cli/command/app/passthrough/Git.java

```diff
@@ -0,0 +1,87 @@
+package bio.terra.cli.command.app.passthrough;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.PassthroughException;
+import bio.terra.cli.exception.UserActionableException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "git", description = "Call git in the Terra workspace.")
+public class Git extends ToolCommand {
+
+  @CommandLine.Option(
+      names = "--resource",
+      description =
+          "name of the git-repo resources in the current workspace to be cloned. \n"
+              + "Example usage: git clone --resource=repo1 --resource=repo2")
+  public String[] names;
+
+  @CommandLine.Option(
+      names = "-a",
```

Comment:
I'd give this a longer name like `--clone-all`.

Reply:
i'd argue for all because the command would be like terra git clone --all and --clone-all would be slightly more repetitive.